pkg/controllers/termination/terminator/eviction.go

```diff
@@ -85,26 +96,26 @@ func (e *EvictionQueue) Add(pods ...*v1.Pod) {
 	}
 }
 
-func (e *EvictionQueue) Start(ctx context.Context) {
-	for {
-		// Get pod from queue. This waits until queue is non-empty.
-		item, shutdown := e.RateLimitingInterface.Get()
-		if shutdown {
-			break
-		}
-		nn := item.(types.NamespacedName)
-		// Evict pod
-		if e.evict(ctx, nn) {
-			e.RateLimitingInterface.Forget(nn)
-			e.Set.Remove(nn)
-			e.RateLimitingInterface.Done(nn)
-			continue
-		}
+func (e *EvictionQueue) Reconcile(ctx context.Context, _ reconcile.Request) (result reconcile.Result, err error) {
+	// Set the result to requeue immediately.
```

Comment:
Don't have a huge problem with it but why reset it to a variable?

Reply:
I had it just to make sure we're doing immediately and doing returned values, but I'll just copy it to the three lines